### PR TITLE
add vc A4

### PR DESCRIPTION
**Pull request by zTgx**
as the title.

### CODE REVIEW

File: tee-worker/litentry/core/credentials/src/templates/a4.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        "shard": "",
+        "name": ""
+    },
+    "subject": "",
+    "issuanceBlockNumber": 0,
+    "expirationBlockNumber": 0,
+    "credentialSubject": {
+        "id": "",
+        "description": "Identity Linked And Verified",
+        "type": "IdentityLinkedVerified",
```
Comment (jingleizhang) [inline tee-worker/litentry/core/credentials/src/templates/a4.json]: please change the above `description` and `type`, e.g, to `Check whether any of the linked accounts hold a minimum amount of LIT NOW` and `LIT Holder`, or something else.

File: tee-worker/litentry/core/assertion-build/src/a4.rs [hunk 2]
```diff
@@ -35,8 +39,19 @@ use std::{
...
+	from_date: ParameterString,
+	min_balance: u128,
+	shard: &ShardIdentifier,
+	who: &AccountId,
+	bn: ParentchainBlockNumber,
+) -> Result<Credential> {
 	let mut client = GraphQLClient::new();
+	let mut flag = false;
+
+	let q_min_balance: f64 = (min_balance / (10 ^ 12)) as f64;
```
Comment (jingleizhang) [inline tee-worker/litentry/core/assertion-build/src/a4.rs]: LIT_TOKEN has multiple decimals: on Eth is 18, and on Substrate is 12. The calculation of q_min_balance may need to be more network related?

Comment (zTgx) [reply tee-worker/litentry/core/assertion-build/src/a4.rs]: what about other networks?

Comment (jingleizhang) [reply tee-worker/litentry/core/assertion-build/src/a4.rs]: Eth & BSC are 18, Litentry&Litmus are 12.

Comment (zTgx) [reply tee-worker/litentry/core/assertion-build/src/a4.rs]: updated

File: tee-worker/litentry/core/credentials/src/templates/a4.json (hunk 1 above)
Comment (zTgx) [reply tee-worker/litentry/core/credentials/src/templates/a4.json]: fixed

File: tee-worker/litentry/core/assertion-build/src/a4.rs [hunk 3]
```diff
@@ -53,6 +67,15 @@ pub fn build(identities: Vec<Identity>, from_date: String, min_balance: f64) ->
 				| VerifiedCredentialsNetwork::Litmus
 				| VerifiedCredentialsNetwork::Ethereum
 		) {
+			let q_min_balance: f64;
+			if verified_network == VerifiedCredentialsNetwork::Litentry
+				|| verified_network == VerifiedCredentialsNetwork::Litmus
+			{
+				q_min_balance = (min_balance / (10 ^ 12)) as f64;
+			} else {
```
Comment (jingleizhang) [inline tee-worker/litentry/core/assertion-build/src/a4.rs:74]: Please make the target network be more specific. 

if verified_network == VerifiedCredentialsNetwork::Litentry || verified_network == VerifiedCredentialsNetwork::Litmus
{
} else if verified_network == VerifiedCredentialsNetwork::Ethereum
{
}else 
{
continue or skip
}

**Review by jingleizhang — APPROVED** [review]
LGTM

**Pull request merged**